### PR TITLE
Update 5.2.5 Release Notes

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Add fixes and known issues for 5.2.5. Also reverse the ordering for known issues per issue #1063

### CODE REVIEW

**Comment by tst-cjeffries** [comment]
https://tst-cjeffries.github.io/earthenterprise/geedocs/5.2.5/answer/7160006.html

File: docs/geedocs/5.2.5/answer/7160006.html [hunk 1]
```diff
@@ -136,6 +141,11 @@ <h5>
             <td>Symlinks are incorrectly world-writable files after installation from RPM or Deb</td>
             <td>Symlinks are now detected during the building of RPMs and Debs and are copied into the package correctly.</td>
           </tr>
+          <tr>
+             <td>1120</td>
+             <td>Installing fusion without server fails because of a missing vars file</td>
```
Comment (tst-nfarah) [inline docs/geedocs/5.2.5/answer/7160006.html]: Since this was introduced and fixed in 5.2.5,  we don't need to document it.

Comment (tst-nfarah) [inline docs/geedocs/5.2.5/answer/7160006.html]: These two new enhancements need to be documented:
https://github.com/google/earthenterprise/issues/1079
https://github.com/google/earthenterprise/issues/1068

See list of resolved issues for 5.2.5:
https://github.com/google/earthenterprise/issues?q=is%3Aissue+milestone%3A5.2.5+is%3Aclosed

Comment (tst-cjeffries) [reply docs/geedocs/5.2.5/answer/7160006.html]: Add a reference to 1079, 1068 was mentioned already in the line about new publish options from the command line. Couldn't find anything else in the list of resolved issues that needed added.

Comment (tst-cjeffries) [reply docs/geedocs/5.2.5/answer/7160006.html]: Removed it.

**Comment by tst-swanson** [comment]
Few minor typos:
When a GEE server is queried for server definitions the response can be returned in proper JSON format. The v parameter can be added to **the** query string to specify which version of the API will be returned. The valid options are v=1 and v=2. Version one returns data in the current format. Version two returns data in JSON format. An example **URL that** returns data in JSON format looks like: http://<GEE earth server>/<database_publish_point>/query?request=Json&v=2.
**Installation** scripts are disabled when Open GEE packages are installed. The install and uninstall scripts will now check if Open GEE has been installed with a package manager before installation begins, and will not make any changes when the packages are installed.
Resolved Issues
1038
genewmapdatabase does not allow flat imagery to be used with mercator **databases**
Known Issues
1102
Should this definitely be REHL6, not RHEL6?
1135
**JavaScript** console error in portable server when loading build date
825
Geserver fails to **start up** fully due to conflicting protobuf library
Throughout
Cap Mercator
Use Earth Client or Earth client
Should Cutter be capped?
I think 2d and 3d should probably be 2D and 3D

**Comment by tst-cjeffries** [comment]
Good catches on all of this. Though the wrong PR must be linked in the JIRA ticket. Ill make sure to fix that.

**Pull request closed without merging**